src/main.py

```diff
@@ -1,372 +1,14 @@
-from config import get_config, SUPPORTED_YEARS, SUPPORTED_LANGUAGES, DEFAULT_YEAR
-from csp import csp
-from feature_policy import feature_policy
-from flask import Flask, abort, redirect, render_template as flask_render_template, request, send_from_directory, url_for
 from flask_talisman import Talisman
-from language import DEFAULT_LANGUAGE, get_language
-import logging
-import random
-from werkzeug.routing import BaseConverter
-from werkzeug.http import HTTP_STATUS_CODES
-from validate import validate
-import os.path
-import re
+from app import create_app, routes, csp, feature_policy
```

Comment:
WDYT about renaming the `app` package to `server`? This will help distinguish it from the client side.

Reply:
Agree, it sounds good